catalogs/sources/gtfs/realtime/unknown-unknown-ilevia-metropole-europeenne-de-lille-gtfs-rt-t-u-1792.json

```
@@ -0,0 +1,9 @@
+{
+    "mdb_source_id": 1792,
+    "data_type": "gtfs-rt",
+    "entity_type": "tu",
```

The `entity_type` should be an array:
```suggestion
    "entity_type": ["tu"],
```